src/helpers/client/safari-client.ts

```diff
@@ -171,14 +197,95 @@ export class SafariClient implements Client {
   }
 
   /**
-   * Sends a keep alive ping every 3 seconds.
+   * Listens for 'AsperaDesktop.Status' events.
+   */
+  private listenStatusEvents() {
+    document.addEventListener('AsperaDesktop.Status', (_: any) => {
```

Comment:
If _ is not used can we just leave it off until it’s used.

Reply:
Done!